doc/source/user_guide/ug_case_settings.rst

```diff
@@ -392,32 +392,39 @@ forcing_nml
    "", "``random``", "wave data file is provided, sea surface height generated using random number (multiple iterations of wave fracture)", ""
    "``ycycle``", "integer", "number of years in forcing data cycle", "1"
    "", "", "", ""
+   
+* = If Icepack is run stand-alone and wave_spec_type is not set to none, then a fixed wave spectrum is defined in the code to use for testing. As with other input data, this spectrum should not be used for production runs or publications.
 
 zbgc_nml
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _tab-bio-tracers-namelist2:
+
 .. csv-table:: **zbgc_nml namelist options**
    :header: "variable", "options/format", "description", "default value"
-   :widths: 15, 15, 30, 15 
+   :widths: 15, 15, 30, 15
 
    "", "", "", ""
    "``algaltype_diatoms``", "real", "mobility type between stationary and mobile algal diatoms", "0.0"
    "``algaltype_phaeo``", "real", "mobility type between stationary and mobile algal phaeocystis", "0.0"
    "``algaltype_sp``", "real", "mobility type between stationary and mobile small plankton", "0.0"
-   "``algal_vel``", "real", ":cite:`Lavoie05`", "1.0e-7"
+   "``algal_vel``", "real", "maximum speed of algae (m/s) :cite:`Lavoie05`", "1.0e-7"
    "``alpha2max_low_diatoms``", "real", "light limitation diatoms 1/(W/m^2)", "0.3"
    "``alpha2max_low_phaeo``", "real", "light limitation phaeocystis 1/(W/m^2)", "0.17"
    "``alpha2max_low_sp``", "real", "light limitation small plankton 1/(W/m^2)", "0.2"
    "``ammoniumtype``", "real", "mobility type between stationary and mobile ammonium", "0.0"
    "``beta2max_diatoms``", "real", "light inhibition diatoms 1/(W/m^2)", "0.001"
    "``beta2max_phaeo``", "real", "light inhibition phaeocystis 1/(W/m^2)", "0.04"
    "``beta2max_sp``", "real", "light inhibition small plankton 1/(W/m^2)", "0.001"
-   "``bgc_flux_type``", "``constant``", "constant ice–ocean flux velocity", "``Jin2006``"
-   "", "``Jin2006``", "ice–ocean flux velocity of :cite:`Jin06`", ""
+   "``bgc_data_type``", "``clim``", "bgc climatological data", "``default``"
```

Comment:
bgc_data_type is in forcing_nml in Icepack and CICE.  Please check the definitions there and remove it here.